static/bidder-info/loyal.yaml

```diff
@@ -0,0 +1,14 @@
+endpoint: "https://us-east-1.loyal.app/pserver"
+maintainer:
+  email: "hello@loyal.app"
```

Comment:
@teqblaze   Email is being sent from prebid team to verify `hello@loyal.app` address. Requesting to respond back on email thread.

Reply:
received response from `loyal` team